packaging_automation/package_cloud_statistics_collector.py

```diff
@@ -40,9 +41,33 @@ class PackageCloudDownloadStats(Base):
     package_full_name = Column(String, nullable=False)
     package_version = Column(String, nullable=False)
     package_release = Column(String)
+    distro_version = Column(String, nullable=False)
+    epoch = Column(String, nullable=False)
     package_type = Column(String, nullable=False)
     download_date = Column(DATE, nullable=False)
     download_count = Column(INTEGER, nullable=False)
+    detail_url = Column(String, nullable=False)
+    UniqueConstraint('package_full_name', 'download_date', "distro_version", name='ux_package_cloud_download_stats')
```

Comment:
I'm tiny bit disturbed that onyl the `distro_version` is in double quotes `"`. Can you make all the same? 😄

Reply:
Fixed